### ISSUE TITLE
UNITAID expenditure is not showing on compare all agencies

### ISSUE THREAD

**Issue by abdulriza** [younginnovations/UN-Transparency-Portal#149]
@bezunakarmi @anjesh UNITAID is currently publishing disbursement and it is capture on their agency page under total expenditure http://open.undg.org/ctrack.html?publisher=XM-DAC-30010&tongue=eng#view=publisher. However, the amount is not shown on the table because they don't publish recipient countries. 

The issue is that disbursement amounts are not shown on the compare all agencies page for UNITAID http://open.undg.org/ctrack.html?tongue=eng#view=publishers. Since UNITAID is using transaction dates, disbursement amounts should be taken for the respective year expenditure in the table. Even though, UNITAID largely publish data prior to 2014, they still have disbursements published for 2014, at least, and they should be shown on the table. Can you have look?

**Comment by bezunakarmi** [comment]
@abdulriza , we have reviewed the above issue. We found out that in the agency page of UNITAID  http://open.undg.org/ctrack.html?publisher=XM-DAC-30010&tongue=eng#view=publisher the expenditure is calculated without using percentage of the  country's budget. The script is calculating the total expenditure irrespective of percentage of country's budget.

But, in the **compare all agencies** page for UNITAID  http://open.undg.org/ctrack.html?tongue=eng#view=publishers, the script uses the country's budget percentage, where as you mentioned above that UNITAID don't publish recipient countries. Therefore the script is ignoring the expenditure of UNITAID in the compare all agencies page.
